lib/smart.js

```diff
@@ -4,14 +4,20 @@ const smartOauthUrl = 'http://fhir-registry.smarthealthit.org/StructureDefinitio
 const { logError } = require('./logging');
 
 /**
- * Get the smart oauth URIs from a Capability Statement
+ * Get the SMART OAuth URIs from a Capability Statement
  *
  * @example
  *
- * oauthUris = authFromCapability(capabilityStatement);
+ * {
+ *   authorizeUrl,
+ *   tokenUrl,
+ *   registerUrl,
+ *   launchRegisterUrl,
```

Comment:
Is this correct? The SMART docs mention a `manage` url, but I don't see a `launch-register` url.
http://docs.smarthealthit.org/authorization/conformance-statement/

Reply:
a google "verbatim" search for "launchRegisterUrl" turns up 0 hits.

Reply:
...same thing with a github "advanced" search, either `extension:rb launchRegisterUrl` or `extension:js launchRegisterUrl`

0 hits, except for  our own repo.

Reply:
^ This should perhaps be its own distinct issue or Trello card; "research".